README.md

```diff
@@ -67,7 +67,7 @@ animation_format = "webp"
 The `[sticker]` section ist optional and can be left out.
 
 `transparent_color` is used as color for semitransparent pixel in `gif`s.
-The field has no effect, if the sticker is not animated or will be converted to `webp` (default).
+The field has no effect, if the sticker is not animated or will be converted to `webp` (default) or gif.
```

Comment:
This isn't exactly true - `transparent_color` **will** have an effect if an animated sticker is converted to gif.